ProjFS.Mac/PrjFSKextTests/MockProc.cpp

```diff
@@ -22,3 +22,43 @@ proc_t vfs_context_proc(vfs_context_t ctx)
 {
     return NULL;
 }
+
```

Comment:
While doing this work, it highlighted 2 more areas to unit test

- CurrentProcessWasSpawnedByRegularUser
- UseMainForkIfNamedStream